homework/style.css

```diff
@@ -1,3 +1,130 @@
 .alert-error {
```

Comment:
Your CSS still needs work for a mobile phone:

![screenshot 2019-01-08 at 19 10 12](https://user-images.githubusercontent.com/2788771/50850105-399be780-1379-11e9-8871-46eca8e1364b.png)